### ISSUE TITLE
provide link shortening with twister's own link shortener

### ISSUE THREAD

**Issue by black-puppydog** [thedod/feed2twister#12]
tasty made a nice, scriptable rpc and corresponding html to shorten links in twister itself. we should use that.

**Comment by slr** [comment]
wrong: RPC is made by @miguelfreitas of course. and you should adopt http://sebsauvage.net/paste/?81e489b51cdf17e3#1Jfo9EhLFdtqufBCWS2VzMRJ4wVXoIbWucHwnIe6ek4= by @homlett.
